indicators/models.py

```diff
@@ -1501,6 +1507,11 @@ def disaggregations(self):
         disaggregations = self.disaggregation.all()
         return self.SEPARATOR.join([x.disaggregation_type for x in disaggregations])
 
+    @property
+    def key_performance_indicator(self):
+        """transition method for key_performance_indicator field removal - may cause unnecessary db calls"""
+        return self.indicator_type.filter(indicator_type="Key Performance Indicator (KPI)").exists()
```

Comment:
I'm wondering if it would make sense to write a couple of tests that cover places where we're using the hardcoded value of "Key Performance Indicator (KPI)" as a type.  If we change the text we use to designate a KPI, this function would return false, even though it is supposed to be true.